examples/em/fdem_ElectricDipoleWholeSpace.py

```diff
@@ -0,0 +1,113 @@
+"""
+Electric Dipole in a Whole Space: Frequency Domain
```

Comment:
@rowanc1, @sgkang, @thast, I would appreciate your thoughts on this interaction with the code and the way the dipoles are set up. It is pretty flexible, and the aim in the code structure is to have it be very interactive and declarative. See what you think!

Reply:
@lheagy nice work! this will be a great place to fill in a number of analytic solutions. Overall, looks pretty good to me. I need some time to play with it. Do you want to refactor em_examples and em_apps with these?

Reply:
Thanks @sgkang! Ideally, yes, we could start to have em_examples rely on this